docs/security-privacy-questionnaire.md

```diff
@@ -0,0 +1,105 @@
+# [Self-Review Questionnaire: Security and Privacy](https://w3ctag.github.io/security-questionnaire/)
+
+> 01.  What information does this feature expose,
+>      and for what purposes?
+
+Payment Link URI is exposed. This includes the scheme (e.g., upi, bitcoin, paypal) and potentially other parameters embedded in the URI, such as payee address, amount, and currency. This information is necessary to identify the payment method and initiate the transaction.
+
+Purposes of Exposing the Information:
+- Payment Initiation: The primary purpose is to allow payment clients (browser extensions or integrated wallets) to detect and handle supported payment methods, streamlining the payment process for users.
```

Comment:
When the word "browser extensions" is used, does that mean https://developer.chrome.com/docs/extensions/reference/api ? Perhaps write something simpler, like "the wallet being used by the user agent". ("User agent" means "browser" in standards language.) The phrase "integrated wallets" also makes sense. 

The "detect" supported payment methods is done by the user agent, I think. Only "handle supported payment methods" is handled by the wallet. Right?

Reply:
That makes sense! Done.